### DISCUSSION TITLE
Qidi X-Plus 3 Installation Help

### DISCUSSION THREAD

**Discussion by TheFace420** [3DCoded/3MS#14] [General Help]
got a xplus 3 i keep getting an error that I cant find 3ms main

**Comment by 3DCoded** [comment]
Does the Qidi X-plus 3 need a Raspberry Pi to print?

If it does need a Pi to print, all the commands should be run over SSH on the **pi**. The 3MS board will be plugged into the **pi**.

If it doesn't need a Pi to print, all the commands should be run over SSH on the **printer**. The 3MS board will be plugged into the **printer**, where you would plug in a webcam. A Raspberry Pi is unnecessary for the 3MS in this case. For the issue about not finding `3ms/main.cfg`, make sure you run the `install.sh` script **on the printer**.
